nodes/config/ui_base.html

```diff
@@ -1467,6 +1679,10 @@
                         buildLayoutOrderEditor(pagesContent)
                         // add Themes View
                         buildThemesEditor(themesContent)
+                        // add Themes View
+                        console.log('sidebar on change')
```

Comment:
🙈

Reply:
It's in Draft - hadn't done a proper sweep myself yet - just wanted to get something pushed up to round the day off